internal/provider/inventory_resource_test.go

```diff
@@ -185,22 +187,17 @@ func TestAccInventoryResource(t *testing.T) {
 			// Create and Read testing
 			{
 				Config: testAccInventoryResourceMinimal(randomName),
-				Check: resource.ComposeAggregateTestCheckFunc(
-					testAccCheckInventoryResourceExists(resourceNameInventory, &inventory),
-					testAccCheckInventoryResourceValues(&inventory, randomName, "", ""),
-					checkBasicInventoryAttributes(t, resourceNameInventory, randomName, "1", "Default"),
-				),
+				Check:  checkBasicInventoryAttributes(t, resourceNameInventory, inventory, randomName, "1", "Default", "", ""),
 			},
-			// Update and Read testing
+			// Update with new org and Read testing
+			{
+				Config: testAccInventoryResourceWithOrg(updatedName, updatedOrgId),
+				Check:  checkBasicInventoryAttributes(t, resourceNameInventory, inventory, updatedName, updatedOrgId, "Default2", updatedDescription, updatedVariables),
```

Comment:
Two points here:

First - Does this mean the org with id `AAP_TEST_ORGANIZATION_ID` needs to have a specific name and the name must be `Default2`? 

If so, we need to call that out in the README - "To run the acceptance tests, you must create an organization called X and set its ID to this env var" - I'd also propose a different name other than `Default2` (there's only one default). Something like `Non-Default` :)

Second - These code changes are modifying an existing acceptance test that was already passing, right? If we're testing a new bug fix, I'd generally prefer to leave an existing test alone and write a new one - unless the test starts failing as a result of the change of course.

The second is not an objection, just a suggestion for next time.

Reply:
Good catch, I didn't include the name in the README, I will change the name and add to the README.

Reply:
Sure. And I agree it's a trade-off and we may be duplicating things, but if we have two tests and one of them starts failing after a future change, we have a pretty good idea of where to look. But if we only have one test that covers lots of things and it fails, we have to look at more code.

Again, not asking to rewrite this, just thinking out loud.

Reply:
That makes sense!

Reply:
I updated the second org name in the test and README.